examples/v1beta1/pipelineruns/alpha/pipeline-object-param-and-result.yaml

```diff
@@ -43,10 +43,22 @@ spec:
     - name: echo-params
       image: bash:latest
       args: [
-        "echo",
-        "--foo=$(params.arg.foo)",
-        "--bar=$(params.arg.bar)"
+        "$(params.arg.foo)",
+        "$(params.arg.bar)"
       ]
+      script: |
+        if [[ $1 != "abc.com" ]]; then
+            echo "Want: abc.com, Got: $1"
+          exit 1
+        else
+          echo "validate the object-foo successfully"
```

Comment:
nit: validated the object param propoerty `arg.foo` successfully

Reply:
Thanks! I will update these